styleguide/_themes/global/scss/fonts.scss

```diff
@@ -159,3 +172,24 @@ i {
 .copyright {
   font-size: .75em;
 }
+
+blockquote {
+  border-left: 0;
+  font-size: 1.75rem;
```

Comment:
+1 for the article @WidgetsBurritos. It was a good read although I skimmed a great deal.

Because of what I feel is some uncertainty behind the use of these units my opinion is that we are best served by using `rem` for `font-size` for the time being. The thinking goes that using `rem` will keep us anchored to the root `font-size` and that all `rem` uses will produce predictable sizes/proportions.

Does that line of thought hold water? 💧 

As the article points out a combination of both units produces good results, and I am all for that when we feel comfortable enough mixing units to produce our desired outcomes.

If that is now, great 😄